src/views/RegistrationView.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+import { ref } from 'vue'
+import { useStore } from 'vuex'
+import { useRouter } from 'vue-router'
+
+const store = useStore()
+const router = useRouter()
+const profileBaseUrl = store.state.ocsApiUrl
```

Comment:
The naming here could potentially be confusing - yeah, we're getting the profile information from the observation portal but an unsuspecting person might be confused as to why it's called profileBaseUrl here. You could get around this by just using the store.state value directly when you build the URL rather than assigning it to a separate variable here.